app/src/main/res/values/strings.xml

```diff
@@ -61,6 +61,8 @@
     <string name="error_wallet_not_found_title">No wallet found</string>
     <string name="error_wallet_not_found_message">Try account or use an another phone number</string>
     <string name="error_too_often_otp_requests_message">You request SMS too often. Try again in %1d sec.</string>
+    <string name="error_shares_does_not_matches_title">Shares aren\'t\nmatch</string>
+    <string name="error_shares_does_not_matches_message">So sorry, baby\n\nIf you will write us, use error code #10464</string>
```

Comment:
error code should be passed from the web3auth

Reply:
you sure ??? damn